cmd/deploy/validate.go

```diff
@@ -23,14 +23,23 @@ type envVar struct {
 	value string
 }
 
-func validateAndSet(variables []string, setEnv func(key, value string) error) error {
-	envVars, err := parse(variables)
+func validateAndSetVariables(opts *Options, setEnv func(key, value string) error) error {
+	cleanEscapedVariables(opts)
```

Comment:
NIT: Can we not modify options, return the variables in a list and assign it later. I think it's more verbose. WDYT

Reply:
done 4e2b7b8ee0f399d8715119761e4b511797e8d6b6